packages/browser-wallet-api-helpers/src/wallet-api-types.ts

```diff
@@ -64,7 +71,7 @@ interface MainWalletApi {
         accountAddress: string,
         type: AccountTransactionType.Update | AccountTransactionType.InitContract,
         payload: SendTransactionPayload,
-        parameters: Record<string, unknown>,
+        parameters: SmartContractParameters,
         schema: string | SchemaWithContext,
```

Comment:
Should these not be optional?

```suggestion
        parameters?: SmartContractParameters,
        schema?: string | SchemaWithContext,
```

Reply:
No, this is one of 2 signatures, below is one where they are not present, which in practice makes them optional. (but in a grouped way)

Reply:
aah... nice..